platform/mds/mds/src/main/java/org/motechproject/mds/builder/impl/MDSConstructorImpl.java

```diff
@@ -65,8 +67,10 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MDSConstructorImpl.class);
 
-    private MdsConfig mdsConfig;
+    @Autowired
     private AllEntities allEntities;
```

Comment:
Why do we initialize one bean on the field level and another on the setter level?